apps/sms/test/marionette/lib/views/shared/tools.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+/* global module */
+
+function Tools() {
+}
```

Comment:
I created a constructor here like in other files, but is it ok to just export a single object containing my method?